### PR TITLE
chore!: Drop support for Node 8

### PR DESCRIPTION
**Pull request by nolanmar511**

### CODE REVIEW

**Comment by nolanmar511** [comment]
PTAL?

File: README.md [hunk 1]
```diff
@@ -7,23 +7,20 @@
...
-    [this](https://bugs.chromium.org/p/v8/issues/detail?id=6623) issue, which
-    causes a memory leak when time profiling is enabled.
-    * Versions of Node.js 10 prior to 10.4.1 are impacted by
-    [this](https://bugs.chromium.org/p/chromium/issues/detail?id=847863) issue,
-    which can cause garbage collection to take several minutes when heap
-    profiling is enabled.
+1. Your application will need to be using Node.js 10.4.1 or greater, or any
+version of Node 12. 
+
+The profiler should not be enabled when using earlier versions of Node 10
```
Comment (macieksmuga) [inline README.md]:
```suggestion
The profiler should not be enabled when using earlier versions of Node 10,
```

File: README.md [hunk 2]
```diff
@@ -7,23 +7,20 @@
...
-    causes a memory leak when time profiling is enabled.
-    * Versions of Node.js 10 prior to 10.4.1 are impacted by
-    [this](https://bugs.chromium.org/p/chromium/issues/detail?id=847863) issue,
-    which can cause garbage collection to take several minutes when heap
-    profiling is enabled.
+1. Your application will need to be using Node.js 10.4.1 or greater, or any
+version of Node 12. 
+
+The profiler should not be enabled when using earlier versions of Node 10 
+versions of Node.js 10 prior to 10.4.1 are impacted by
```
Comment (macieksmuga) [inline README.md]:
```suggestion
since versions of Node.js 10 prior to 10.4.1 are impacted by 
```

File: system-test/system_test.sh [hunk 3]
```diff
@@ -35,7 +35,7 @@ for i in ${NODE_VERSIONS[@]}; do
 
   # Test support for accurate line numbers with node versions supporting this
   # feature.
-  if [ "$i" != "8" ] && [ "$i" != "10" ] && [ "$i" != "11" ]; then
+  if [ "$i" != "10" ] && [ "$i" != "11" ]; then
```
Comment (macieksmuga) [inline system-test/system_test.sh]: Does Node 12 support this?

Comment (nolanmar511) [reply system-test/system_test.sh]: Node 12 and the V8 canary version support accurate line numbers, but node 8, 10, and 11 did not support this.

**Comment by nolanmar511** [comment]
I'm going to spend a bit of time attempting to fix Windows tests. If I fix the tests on windows, I may try to release pprof-nodejs before merging this in, since it will allow us to fix windows unit tests for cloud-profiler-nodejs before dropping Node 8.

**Pull request closed without merging**